### DISCUSSION TITLE
Button not labeled correctly, marked with ??? question marks

### DISCUSSION THREAD

**Discussion by aetherfaerber** [kitodo/kitodo-production#6296] [Q&A] [answered]
Hello again

Since we have started using Kitodo the "renameMediaFiles"-button in the metadata editor has always looked like this:

![image](https://github.com/user-attachments/assets/8554d746-7e83-4ed9-95a3-b1b34ad32e71)

According to https://github.com/kitodo/kitodo-production/blob/d074f6182815406e16bd4ed4122c0c021c3f8577/Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/header.xhtml#L67 and https://github.com/kitodo/kitodo-production/blob/d074f6182815406e16bd4ed4122c0c021c3f8577/Kitodo/src/main/resources/messages/messages_en.properties#L996 it should be labeled "Rename media". Translations are also available.

Is everyone affected by this?

**Comment by solth** [comment, marked as answer]
I haven't seen this problem for this particular button. Which release do you use? Do you use custom messages in your `/usr/local/kitodo/messages/` folder? Than these are used instead of those provided by the application itself and you need to update your custom message files whenever their contents change in the repository or in a new release. Maybe you didn't add the new messages keys that came with the "Rename media" feature?

My advice would be to leave the message configuration folder mentioned above empty if you do not absolutely need custom messages and keys to appear in the application, it makes maintenance much easier.

**Reply by aetherfaerber** [reply]
Thank you for the prompt and very useful answer! This solved the problem.
